.github/workflows/test-composite-actions.yaml

```
@@ -77,3 +77,65 @@ jobs:
         with:
           rosdistro: ${{ matrix.rosdistro }}
           target-packages: empty_target_cmake empty_target_python
+
+  test-file-check:
```

Please replace all the words `file-check`.

```suggestion
  test-check-file-existence:
```